<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>575</width>
    <height>451</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>BD</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="page_2">
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="1" column="0">
         <widget class="QTabWidget" name="tabWidget">
          <property name="currentIndex">
           <number>1</number>
          </property>
          <widget class="QWidget" name="tab">
           <attribute name="title">
            <string>Поик в базе</string>
           </attribute>
           <layout class="QGridLayout" name="gridLayout_4">
            <item row="0" column="0">
             <widget class="QLineEdit" name="stringInput"/>
            </item>
            <item row="0" column="2">
             <widget class="QPushButton" name="pushButton_2">
              <property name="text">
               <string>DO IT</string>
              </property>
             </widget>
            </item>
            <item row="2" column="0" colspan="3">
             <widget class="QTableWidget" name="tableWidget"/>
            </item>
            <item row="0" column="1">
             <widget class="QSpinBox" name="numberInput">
              <property name="minimumSize">
               <size>
                <width>240</width>
                <height>23</height>
               </size>
              </property>
              <property name="maximum">
               <number>999</number>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="tab_2">
           <attribute name="title">
            <string>Просмотр VSAM (строки)</string>
           </attribute>
           <layout class="QGridLayout" name="gridLayout_5">
            <item row="1" column="1">
             <widget class="QSpinBox" name="StrRegion">
              <property name="maximum">
               <number>999</number>
              </property>
             </widget>
            </item>
            <item row="1" column="3">
             <widget class="QSpinBox" name="StrInter">
              <property name="maximum">
               <number>999</number>
              </property>
             </widget>
            </item>
            <item row="1" column="4">
             <widget class="QPushButton" name="pushButton_3">
              <property name="text">
               <string>Показать</string>
              </property>
             </widget>
            </item>
            <item row="3" column="0" colspan="5">
             <widget class="QTableWidget" name="stringTable"/>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="label_2">
              <property name="text">
               <string>Управляемая область</string>
              </property>
             </widget>
            </item>
            <item row="1" column="2">
             <widget class="QLabel" name="label_3">
              <property name="text">
               <string>Интервал</string>
              </property>
             </widget>
            </item>
            <item row="0" column="0" colspan="5">
             <widget class="QLabel" name="TotalRegionsStr">
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="tab_3">
           <attribute name="title">
            <string>Просмотр VSAM (числа)</string>
           </attribute>
           <layout class="QGridLayout" name="gridLayout_6">
            <item row="1" column="0">
             <widget class="QLabel" name="label_5">
              <property name="text">
               <string>Управляемая область</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QSpinBox" name="numRegion">
              <property name="maximum">
               <number>999</number>
              </property>
             </widget>
            </item>
            <item row="1" column="2">
             <widget class="QLabel" name="label_4">
              <property name="text">
               <string>Интервал</string>
              </property>
             </widget>
            </item>
            <item row="1" column="3">
             <widget class="QSpinBox" name="numInter">
              <property name="maximum">
               <number>999</number>
              </property>
             </widget>
            </item>
            <item row="1" column="4">
             <widget class="QPushButton" name="pushButton_4">
              <property name="text">
               <string>Показать</string>
              </property>
             </widget>
            </item>
            <item row="0" column="0" colspan="5">
             <widget class="QLabel" name="TotalRegionsNum">
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
            <item row="2" column="0" colspan="5">
             <widget class="QTableWidget" name="numTable"/>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="page">
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="1">
         <spacer name="verticalSpacer_3">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>59</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="1" column="0">
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>246</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="1" column="1">
         <widget class="QLabel" name="label">
          <property name="text">
           <string>TextLabel</string>
          </property>
         </widget>
        </item>
        <item row="1" column="2">
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>245</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="2" column="1">
         <spacer name="verticalSpacer_2">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>172</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="3" column="1">
         <widget class="QPushButton" name="pushButton">
          <property name="text">
           <string>RUN</string>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>112</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>tabWidget</tabstop>
  <tabstop>stringInput</tabstop>
  <tabstop>numberInput</tabstop>
  <tabstop>pushButton_2</tabstop>
  <tabstop>tableWidget</tabstop>
  <tabstop>StrRegion</tabstop>
  <tabstop>StrInter</tabstop>
  <tabstop>pushButton_3</tabstop>
  <tabstop>numInter</tabstop>
  <tabstop>pushButton</tabstop>
  <tabstop>stringTable</tabstop>
  <tabstop>numRegion</tabstop>
  <tabstop>pushButton_4</tabstop>
  <tabstop>numTable</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
